packages/server/package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@botpress/messaging-base": "*",
     "@botpress/messaging-channels": "*",
+    "@botpress/messaging-channels-legacy": "npm:@botpress/messaging-channels@0.1.0",
     "@botpress/messaging-engine": "*",
```

Comment:
That's how we are going to use two versions of each channel at the same time using this dependency alias